### PR TITLE
definitions: Changed scale_factor of kilogram

### LINKED ISSUE #14591: magnetic_constant does not evaluate to the correct number
Hello!

I found that when importing u0 or the magnetic_constant from sympy.physics.units in sympy 1.1.1, evaluating the constant is 3 sig figs off.

```
>>> from sympy.physics.units import u0
>>> u0.args
(magnetic_constant, Dimension(force/current**2), pi/2500, magnetic_constant)
>>> u0.scale_factor.evalf()
0.00125663706143592
```

The actual value of u0 should be pi/2500000 (0.00000125663706143592) like it evaluates to in version 1.0.

### PR DESCRIPTION
**Pull request by lilithxxx**
```kilogram``` has now ```scale_factor``` 'One' instead of ```gram```. This lets ```kilogram``` to be the base unit.
#### References to other Issues or PRs

Fixes #14591

### CODE REVIEW

File: sympy/physics/units/tests/test_unitsystem.py [hunk 1]
```diff
@@ -56,7 +56,7 @@ def test_print_unit_base():
         Js.set_scale_factor(S.One)
 
         mksa = UnitSystem((m, kg, s, A), (Js,))
-        assert mksa.print_unit_base(Js) == m**2*kg*s**-1/1000
+        assert mksa.print_unit_base(Js) == m**2*kg*s**-1
```
Comment (moorepants) [inline sympy/physics/units/tests/test_unitsystem.py]: You'll need to deprecate this behavior if you want to change it.

**Comment by Upabjojr** [comment]
I don't see the purpose of this PR. Scale factors are internal values and users should only interact with the division of two scale factors.

I suggest closing this PR.

**Comment by lilithxxx** [comment]
This was related to #14591. If ```kilogram``` is not given the scale_factor One then the answer is not correct. Moreover this change seems right as it is the S.I unit and not ```grams```

**Comment by Upabjojr** [comment]
`scale_factor` should be abolished.

**Pull request closed without merging**